relbench/tasks/event.py

```diff
@@ -55,6 +50,9 @@ def make_table(self, db: Database, timestamps: "pd.Series[pd.Timestamp]") -> Tab
             GROUP BY
                 t.timestamp,
                 event_attendees.user_id
+                ) event_info
+            WHERE
+                prev_target IS NULL OR prev_target > 0
```

Comment:
why do we include `prev_target IS NULL`?

Reply:
OMG. Thanks for the detailed reviewing. Sorry I was trying to finish this pr too quick and didn't notice the error in the sql.

Reply:
Actually if I fix the issue here the training rows would be reduced from 20k to 2000. We decided to drop the temporal target filter.